src/DwitTech.AccountService.Core/Extensions/ServiceCollectionExtension.cs

```diff
@@ -21,10 +21,16 @@ public static IServiceCollection AddDatabaseService(this IServiceCollection serv
         {
 
             string connectionString = configuration.GetConnectionString("AccountDbContext");
-            connectionString = connectionString.Replace("{DBHost}", configuration["DB_HOSTNAME"]);
-            connectionString = connectionString.Replace("{DBName}", configuration["DB_NAME"]);
-            connectionString = connectionString.Replace("{DBUser}", configuration["DB_USERNAME"]);
-            connectionString = connectionString.Replace("{DBPassword}", configuration["DB_PASSWORD"]);
+            //connectionString = connectionString.Replace("{DBHost}", configuration["DB_HOSTNAME"]);
+            //connectionString = connectionString.Replace("{DBName}", configuration["DB_NAME"]);
+            //connectionString = connectionString.Replace("{DBUser}", configuration["DB_USERNAME"]);
+            //connectionString = connectionString.Replace("{DBPassword}", configuration["DB_PASSWORD"]);
+
+            connectionString = connectionString.Replace("{DBHost}","localhost");
```

Comment:
secrets in the code!!! Yes i know it is a dummy, please use usersecrets or gitignored appsettings.local.json